### DISCUSSION TITLE
Commonly Misspelled Words List

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1104] [Ideas] [answered]
It would be nice to have a list of words generated through tests you take that show you which words you made a mistake on the most. Then you would be able to take a custom test with those words.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
I think there's an issue already made for that which is #44

It's been here for quite some time (basically since the creation of Monkeytype or at least the github) so I don't know if it's gonna come out or not even though there were some people working on it
